helm/applications/base/values.yaml

```diff
@@ -1,6 +1,11 @@
 kubernetesClusterDomain: cluster.local
+
+# The Namespace of the APIs (UIs and APIs)
 skaha:
   namespace: skaha-system
+
```

Comment:
If you feel its worth it, maybe format doclines in valyes file in the following format: https://github.com/norwoodj/helm-docs which in the future can lead to automatically generated readme with helm-docs.